widgets/DiscordappFrame.php

```diff
@@ -1,9 +1,11 @@
 <?php
+
 namespace humhub\modules\discordapp\widgets;
 
+use humhub\models\Setting;;
 use humhub\components\Widget;
 
-class DiscordappFrame extends Widget
+class Discordapp extends \humhub\components\Widget
```

Comment:
I have plans for this rework, as it stands the module would just be a copy of your module, I'm working out from where you update. Once the webhook is done everything will work fine as well.